hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -19,7 +19,7 @@
 This is a full example hazelcast.xml that includes all the configuration elements and attributes of Hazelcast.
 To use this, rename it to hazelcast.xml and place it in the directory where you start Hazelcast.
 Please see the schema to learn how to configure Hazelcast at 
-https://hazelcast.com/schema/config/hazelcast-config-3.9.xsd or the Reference Manual at
+https://hazelcast.com/schema/config/hazelcast-config-3.10.xsd or the Reference Manual at
```

Comment:
You need to add an example for flake IDs in this file, as well as the documentation.